integration-tests/src/test/java/org/zowe/apiml/integration/proxy/ServerSentEventsProxyTest.java

```diff
@@ -76,50 +73,41 @@ void givenValidSsePaths_thenReturnEvents() {
         @Nested
         class GivenIncorrectPath_thenReturnError {
             @Test
-            @Disabled("FIXME: fix messaging to be same as in the original ZUUL Gateway")
             void givenInvalidServiceId() {
                 String path = "/bad/sse/v1/events";
-                FluxExchangeResult<String> fluxResult = webTestClient
+                webTestClient
                     .get()
                     .uri(path)
                     .exchange()
                     .expectStatus()
                     .isNotFound()
                     .returnResult(String.class);
```

Comment:
We are not returning any error codes in these responses now?
If we are, can we add the assertions back on the content?